challenge_5/python/mjuiuc/test.py

```diff
@@ -47,8 +47,8 @@ def test_case5(self):
 		t1 = time.time()
 		case5out = case5in.findTheDifference( finalTestStringS, finalTestStringT + 'p')
 		t2 = time.time()
-		print 'Runtime of Case5: ' + str(t2-t1) + ' seconds' 
+		print('Runtime of Case5: ' + str(t2-t1) + ' seconds')
 		self.assertEqual( case5out, 'p')
 
 if __name__ == '__main__':
-	unittest.main()
```

Comment:
It seems the difference between these two lines is a whitespace at the end? This was not an intended change, but I don't think it matters either way.